src/App.jsx

```diff
@@ -7,16 +7,20 @@ import TimerArea from './component/TimerArea';
 import Footer from './component/Footer';
 import Header from './component/Header';
 
-const App = () => (
-  <div className="d-flex flex-column min-vh-100">
-    <Header />
-    <Container fluid className="my-auto">
-      <Row className="mt-3">
-        <Col md={{ span: 6, offset: 3 }}><TimerArea /></Col>
-      </Row>
-    </Container>
-    <Footer />
-  </div>
-);
+const App = () => {
+  const [startAuto, setStartAuto] = useState(false);
+  const handelStartAuto = () => { setStartAuto(!startAuto); };
```

Comment:
You should rename `handelAutoStart` -> `toggleShouldAutoStart`

The name `toggleShouldAutoStart` implies that this function flips the
value of shouldAutoStart, which is exactly what the function does.
However, the name prefix `handle` is generic and doesn't give a clear
meaning of what the function does.